src/control/cmd/dmg/system.go

```diff
@@ -106,7 +106,9 @@ type systemQueryCmd struct {
 	ctlInvokerCmd
 	cmdutil.JSONOutputCmd
 	rankListCmd
-	Verbose bool `long:"verbose" short:"v" description:"Display more member details"`
+	Verbose      bool   `long:"verbose" short:"v" description:"Display more member details"`
+	NotOK        bool   `long:"not-ok" description:"Display components in need of administrative investigation"`
+	WantedStates string `long:"with-states" description:"Only show engines in one of a set of states specified as a comma separated list of AwaitFormat Starting Ready Joined Stopping Stopped Excluded AdminExcluded Errored Unresponsive"`
```

Comment:
nit: I don't think it's necessary to include all of the states in the description... I understand the intent to be helpful, but it makes the help output pretty verbose and is also a maintenance burden.

The gold-plated solution here is to make `WantedStates` a custom type that implements the go-flags [Completer](https://pkg.go.dev/github.com/jessevdk/go-flags#Completer) interface so that the admin can tab-complete with a programmatically-generated list of state names.

The simpler solution is to just refer the reader to the relevant section of the admin guide, e.g. "Display members matching the comma-separated state list (refer to Member States in the admin guide for details)". If that content doesn't exist, it's probably a good idea to add it. 😉